webapp/app/model/form_data.py

```diff
@@ -157,6 +157,18 @@ def account_holder_must_be_declared(cls, v, values):
                 raise MissingError
         return v
 
+    @validator('person_a_requests_pauschbetrag', always=True)
+    def person_a_required_has_disability(cls, v, values):
+        if not values.get('person_a_has_disability') == 'yes' and not v:
+            raise MissingError
+        return v
```

Comment:
I don't think this is the right way around? We want
```
if values.get('person_a_has_disability') == 'yes' and not v:
    raise MissingError
return v
```

Reply:
And please add a test to cover this.

Reply:
I'm confused... When the `person_a_has_disability` is yes then the model should be fine? if the value is no then the person_a_requests_pauschbetrag is not mandatory or? Am I getting this completely wrong? :confused:

Reply:
Actually, my head was in the completely wrong place when I wrote that comment alltogether. has_disability and requests_pauschbetrag have no direct logic between them. They are dependent on the merkzeichen page. So the validators don't make any sense. What we need are validators that check that the person has a Pauschbetrag > 0. But I'm already doing that in my new PR, so maybe just delete then here and I'll propose a way to change this in the other PR?